vectordatasource/transform.py

```diff
@@ -9472,9 +9484,33 @@ def apply_disputed_boundary_viewpoints(ctx):
         # we want to apply disputes to already generally-unrecognised borders
         # too, as this allows for multi-level fallback from one viewpoint
         # possibly through several others before reaching the default.
-        elif (kind.startswith('unrecognized_') and
-              kind[len('unrecognized_'):] in _BOUNDARY_KINDS):
-            boundaries.append((shape, props, fid))
+        elif kind.startswith('unrecognized_'):
+            if kind[len('unrecognized_'):] in _BOUNDARY_KINDS:
+                boundaries.append((shape, props, fid))
+
+        # this covers the disputed_reference_line case.
+        # unpack all the viewpoints from disputed_by, and make them all unrecognized
+        # TODO what do I need to do with geometry here?
+        elif kind == 'disputed_reference_line':
```

Comment:
Modulo other PR comments and IRL discussion update `disputed_reference_line` to `disputed`?

Reply:
Both of these are valid kinds now, so evaluate if you need to extend this logic.